include/pnet_api.h

```diff
@@ -465,20 +465,6 @@ typedef struct pnet_result
    uint16_t                add_data_2;    /**< Application-specific error information. */
 } pnet_result_t;
 
-/**
```

Comment:
Is this information (header files) incorrect or obsoleted by the documentation above?

Reply:
This additional text field is confusing Doxygen (it is not connected to a function etc). The info is moved to to the freetext part of the documentation.

Reply:
Modified so that it not will confuse Doxygen.